content/developer/reference/backend/views.rst

```diff
@@ -305,8 +311,9 @@ how the matched node should be altered:
         <attribute name="attrs">
           {'invisible': [('sale_ok', '=', False)], 'readonly': [('editable', '=', False)]}
         </attribute>
+        <attribute name="class" remove="mt-2" separator=" "/>
```

Comment:
I see you added this line. Is it correct to specify the `separator` attribute for `remove`? If you do need to remove `separator`, please also swap this line with the one showcasing `add` so we follow the specs order.

Reply:
You need separator for both add and remove
https://github.com/odoo/odoo/blob/be43710ab8da3be5677ae4cd3fd2e3dbe1be637b/odoo/tools/template_inheritance.py#L198-L202

But `add` and `remove` are also often used together
https://github.com/odoo/odoo/blob/74a594bb2e68662b22477fcce24d9620b2dd0b2a/addons/website/views/website_templates.xml#L136

Reply:
Ok! It's indeed clearer with a bit of code :p

It appears I misinterpreted what @poma-odoo originally wrote ("the values given by `add` is added to the values separated by `separator`") as "...added to the values _**,**_ separated by `separator`". The difference is subtle but it's an entirely different story, even if `separator` is also used to append `add`'s values.

I will rephrase this and push a new version to clarify it. Let's make it crystal clear.